### PR TITLE
Guidance (Follow Ups): General UI

### PR DESCRIPTION
**Pull request by mxosman**
## Description of the change

Addresses general UI feedback:
* Metric Configuration vertical responsiveness (should still be able to see and scroll through all metrics on the list)
* Adjust flashing of 0 of X metrics before being replaced by the right number (added a loader to the container)
* Adjust alignment of Onboarding screen (adjusted per the new Figma [design](https://www.figma.com/file/dvX3yg7FMGJSWFaqHWrWgQ/Justice-Counts-Publisher?node-id=6094%3A14587&t=KRytwPgZ2chhIBYR-1))
* Reduce spacing between “Skip” and button in Agency Setup (reduced by 16px)
* Update Records page copy to “No records to display. Create a record?”
* Update all Guidance copy to replace “Report” with “Record”

https://user-images.githubusercontent.com/59492998/219802960-323d6dac-3137-4b5b-9c3e-cec464ed81d2.mov

NOTE: I don't have a demo for the copy updates because this whole feature needs to be rebased with main to capture all the new updates in the BE - right now the Records page is crashing because the update to user roles.

## Related issues

Contributes to #329 

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: publisher/src/components/Guidance/Guidance.tsx [hunk 1]
```diff
@@ -274,69 +274,75 @@ export const Guidance = observer(() => {
...
-            <MetricListContainer>
-              {metricsEntries.map(([systemMetricKey, metric]) => {
-                const metricCompletionProgress =
-                  getOverallMetricProgress(systemMetricKey);
-                const metricCompletionProgressValue =
-                  getMetricCompletionValue(systemMetricKey);
-                const { system, metricKey } =
-                  MetricConfigStore.splitSystemMetricKey(systemMetricKey);
+            {totalMetrics === 0 ? (
+              <Loader />
```
Comment (lilidworkin) [inline publisher/src/components/Guidance/Guidance.tsx:278]: Nice!

File: publisher/src/components/Guidance/types.ts [hunk 2]
```diff
@@ -59,8 +61,7 @@ export const onboardingTopicsMetadata: OnboardingTopicsMetadata = {
   },
   ADD_DATA: {
     displayName: "Add data",
-    description:
-      "You can now upload data to your metrics by filling out a monthly report or by uploading a spreadsheet.",
+    description: `You can now upload data to your metrics by filling out a monthly ${REPORTS_LOWERCASE} or by uploading a spreadsheet.`,
```
Comment (lilidworkin) [inline publisher/src/components/Guidance/types.ts:64]: Should this be singular REPORT_LOWERCASE?

**Review by lilidworkin — APPROVED** [review]
LGTM!

File: publisher/src/components/Guidance/types.ts (hunk 2 above)
Comment (mxosman) [reply publisher/src/components/Guidance/types.ts:64]: Nice catch!

**Pull request merged**